src/mon/Monitor.cc

```diff
@@ -4844,16 +4846,19 @@ bool Monitor::_scrub(ScrubResult *r,
     }
 
     bufferlist bl;
-    //TODO: what when store->get returns error or empty bl?
-    store->get(k.first, k.second, bl);
+    int err = store->get(k.first, k.second, bl);
+    assert(err == 0 && bl.length() > 0);
```

Comment:
@tchaikov I don't understand this comment. Could you please explain more ? In assert we are checking if bl is empty or not.